### ISSUE TITLE
“smoker”-inclusions in macrophages

### ISSUE THREAD

**Issue by mgriese** [obophenotype/human-phenotype-ontology#5557]
**Preferred term label:**
“smoker”-inclusions in macrophages

**Synonyms**

**Definition (free text, please give PubMed ID)**
In otherwise healthy smokers, characteristic so called “smoker-inclusion” can be found within the macrophages in the bronchoalveolar fluid. These blue/ black/ round/ oval cytoplasmic inclusions consist of pigmented lipid deposits.
Costabel U. Atlas of bronchoalveolar lavage. London: Chapman & Hall; 1998. ISBN0412792702.

**Parent term (use hpo.jax.org/app)**
Abnormalities of alveolar macrophages

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
@GrieseAG
@Katrinknoflach

**Comment by pnrobinson** [comment]
added term
